integration-tests/src/test/register-address-v2.test.ts

```diff
@@ -68,16 +68,9 @@ describe('RegisterAddressV2', () => {
     });
 
     it('registerAddressV2 PersonalSign works as expected', async (): Promise<void> => {
-        const {
-            api,
-            extrinsics: { registerAddressV2 },
-        } = ccApi;
-
         const lenderWallet = Wallet.createRandom();
-        const accountId = await personalSignAccountId(api, lenderWallet, lender.addressRaw);
-        const proof = personalSignSignature(accountId);
 
-        const lenderRegAddr = await registerAddressV2(lenderWallet.address, blockchain, proof, lender);
+        const lenderRegAddr = await registerAddressV2Example(ccApi, lenderWallet, lender, blockchain);
```

Comment:
Ahhh, we don't want to do that. This needs 2 test scenarios:

1) exercise the creditcoin-js extrinsics wrapper because this is the actual API interface and it is what other developers will be using. And then:
2) exercise the example separately as a sanity check to make sure that the code is still relevant.

Reply:
Ah, I see. The `loan-example` call uses the example function as the main integration test and I just copied that pattern. Will update it now